### DISCUSSION TITLE
Information and help for an ultra-minimalist desktop: only a few buttons and a minimum of firefox and other software

### DISCUSSION THREAD

**Discussion by MezaLab** [lxqt/lxqt#2305] [General]
Hello

For a particular project intended then of many users our NPO / NGOrganisation seeks information and especially of the assistance so that is carried out for Debian an ultra minimalist desktop, which would comprising:

- A wallpaper 
- Buttons for Stop and Restart 
- Possibly some functions like audio and video settings
- At least, a Firefox stripped down to the max
- Possibly the client/viewer of an open, decentralized virtual reality, etc.

Nothing else should be accessible to the user not even a command line client, etc!

It seems that LXQT allows to realize such a desktop!
Is this the case? 

We don't have any competent person to do it and we don't have any financial means at the moment! On the other hand, the project should eventually create jobs and promote the development of free software.

If you express yourself in French it would be better ;)

Thanks

**Comment by stefonarch** [comment]
Somehow I oversaw that. Some thing you'd need:

* configure autologin for that user, another admin user, and also "close session" button.
* block `~/.config/lxqt/panel.conf` once configured all,  using `chattr +i`
* no desktop module, no runner, no globalkeys module, for the wallpaper another solution is needed.

Maybe there are other things, it needs testing.

**Comment by giang61** [comment]
Thank you, Stefonarch!  I'll try to implement what you suggested and will let you know asap!  Best,
